packages/next-urql/src/__tests__/init-urql-client.spec.ts

```diff
@@ -1,63 +1,21 @@
-import {
-  ssrExchange,
-  debugExchange,
-  dedupExchange,
-  cacheExchange,
-  fetchExchange,
-} from 'urql';
-
 import { initUrqlClient } from '../init-urql-client';
 
 describe('initUrqlClient', () => {
   it('should return the urqlClient instance and ssrCache', () => {
-    const [urqlClient, ssrCache] = initUrqlClient({
+    const urqlClient = initUrqlClient({
       url: 'http://localhost:3000',
     });
 
     expect(urqlClient).toHaveProperty('url', 'http://localhost:3000');
     expect(urqlClient).toHaveProperty('suspense', true);
-    // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    expect(ssrCache!.toString()).toEqual(ssrExchange().toString());
   });
 
   it('should accept an optional mergeExchanges function to allow for exchange composition', () => {
```

Comment:
Looks like we can fully yeet away this test now that this is all being initialized in a single function 🎉